### PR TITLE
add visited pins view

### PR DESCRIPTION
**Pull request by sokolikp**
visisted-view

### CODE REVIEW

File: Sweeper/View Controllers/PinsMapViewController.swift [hunk 1]
```diff
@@ -54,18 +54,6 @@ class PinsMapViewController: UIViewController {
             }
         }
     }
-
-    @IBAction func onPost(_ sender: UIBarButtonItem) {
```
Comment (huw30) [inline Sweeper/View Controllers/PinsMapViewController.swift:58]: Did you move this to somewhere else?

**Review by huw30 — APPROVED** [review]

File: Sweeper/View Controllers/PinsMapViewController.swift (hunk 1 above)
Comment (sokolikp) [reply Sweeper/View Controllers/PinsMapViewController.swift:58]: We only access the "Post" from the tab bar controller now.

Comment (huw30) [reply Sweeper/View Controllers/PinsMapViewController.swift:58]: oh, right! LGTM

**Review by wumingx89 — APPROVED** [review]
LGTM
🚢

**Pull request merged**